examples/tracing/tracing.bal

```diff
@@ -0,0 +1,44 @@
+import ballerina/http;
+import ballerina/log;
+import ballerina/runtime;
+import ballerina/observe;
+
+// Make sure you start the service with --observe, or tracing enabled.
+service<http:Service> hello bind { port: 9090 } {
+
+    // Invoke all resources with arguments of server connector and request.
+    sayHello(endpoint caller, http:Request req) {
+        http:Response res = new;
+
+        //Start a child span attaching to the system span generated.
+        int spanId = check observe:startSpan("MyFirstLogicSpan");
+
+        //Start a new Root span without attaching to the system span.
+        int rootParentSpanId = observe:startRootSpan("MyRootParentSpan");
+        // Some actual logic will go here, and for example we have introduced some delay with sleep.
+        runtime:sleep(1000);
+        //Start a new child span for the span `MyRootParentSpan`.
+        int childSpanId = check observe:startSpan("MyRootChildSpan", parentSpanId = rootParentSpanId);
+        // Some actual logic will go here, and for example we have introduced some delay with sleep.
```

Comment:
Can we use a different example so that we are showcasing more to the user?

Reply:
If there are irrelevant code segments for tracing, then the whole objective of demonstrating the tracing API will be reduced. And for the intended cases, sleep will be the shortest and easiest way.